### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by damiansteiger**

### CODE REVIEW

**Review by babbush — DISMISSED** [review]

**Comment by babbush** [comment]
LGTM. I'll let you merge when ready in case you wanted Thomas to look at it for some reason.

**Comment by damiansteiger** [comment]
Thanks. 
Yes, Thomas is the master of setup scripts and should double check ;)

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import os
```
Comment (thomashaener) [inline setup.py]: Do we need this?

**Review by thomashaener — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (damiansteiger) [reply setup.py]: removed

**Review by thomashaener — APPROVED** [review]

**Pull request merged**